CHANGELOG.md

```diff
@@ -16,8 +16,13 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Push Notification uses `MessagingStyle` on devices with API Version 23+
+- Push Notification configuration has been simplified, check our [docs](https://getstream.io/chat/docs/sdk/android/client/guides/push-notifications/#customizing-push-notifications) to see how it works
+- `NotificationHandler` interface allows you to implement your own Push Notification logic show/remove notifications
```

Comment:
Would be nice to mention that `NotificationHandler` is a replacement for `ChatNotificationHandler` and we changed the way for initializing notifications in `ChatClient` (they need to pass `NotificationConfig` instead of whole `NotificationHandler` and they need to override `NotificationHandlerFactory.newIntent`)